assembly/src/assembler/instruction/procedures.rs

```diff
@@ -5,118 +5,29 @@ use super::{Assembler, BasicBlockBuilder, Operation};
 use crate::{
     assembler::{mast_forest_builder::MastForestBuilder, ProcedureContext},
     ast::{InvocationTarget, InvokeKind},
-    AssemblyError, RpoDigest, SourceSpan, Spanned,
+    AssemblyError, RpoDigest,
 };
 
 /// Procedure Invocation
 impl Assembler {
+    /// Returns the [`MastNodeId`] of the invoked procedure specified by `callee`.
+    ///
+    /// For example, given `exec.f`, this method would return the procedure body id of `f`.
     pub(super) fn invoke(
```

Comment:
Should we also mention in the comments that this may insert an `External` node into the `MastForest` if one is missing?

Reply:
done